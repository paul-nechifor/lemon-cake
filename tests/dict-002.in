(dict (quote a) 1 (quote bb) 8 (quote ccc) 123)
