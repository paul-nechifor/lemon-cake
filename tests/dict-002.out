(dict bb 8 ccc 123 a 1)
